src/pages/community-cloud.js

```diff
@@ -100,14 +101,12 @@ const CommunityCloudPage = () => {
                     </SimpleGrid>
                 </Card>
 
-                <Title order={2} my="md">Workloads</Title>
+                <ODHServices />
```

Comment:
it's not just ODH services. Let's just call it `Services`

Reply:
@bryanmontalvan I agree with this, let's replace mention of odh with just services (including `odh.js`). ODH is one of many services.